Extensions/blacklist.js

```diff
@@ -124,6 +124,10 @@ XKit.extensions.blacklist = new Object({
 
 	run: function() {
 		this.running = true;
+		XKit.css_map.add_callback('blacklist', this.actuallyRun);
+	},
+
+	actuallyRun: function() {
```

Comment:
im pretty sure theres nothing that relies on Blacklist running synchronously, so it would be much nicer to just do `run: async function() {` and await the CSS map